addons/sale_loyalty/models/sale_order.py

```diff
@@ -175,6 +190,20 @@ def action_open_reward_wizard(self):
             return True
         return self.env['ir.actions.actions']._for_xml_id('sale_loyalty.sale_loyalty_reward_wizard_action')
 
+    def action_view_gift_cards(self):
```

Comment:
IMO it would better to create XML action whenever it is possible as there are some issues with python actions :see_no_evil: 

I don't have very strong opinion on that. @Feyensv can you check which would be better?